docs/docs/build/getting-started/helloworld.mdx

```diff
@@ -44,91 +51,419 @@ Follow the steps from the [foundry book](https://book.getfoundry.sh/getting-star
   </TabItem>
 </Tabs>
 
-### Creating the Coin Contract
+### Creating the HelloBitcoin Contract
 
 <Tabs groupId="dev-env">
   <TabItem value="remix" label="Remix">
 
-Create a new project with Remix. Under `contracts` folder create a new file `Coin.sol`.
+Create a new project with Remix. Under `contracts` folder create a new file `HelloBitcoin.sol`.
+
+Add the following remappings in `compiler_config.json` under `settings`
+
+```json
+{
+  "language": "Solidity",
+  "settings": {
+      "optimizer": {
+          "enabled": true,
+          "runs": 200
+    },
+    "outputSelection": {
+          "*": {
+          "": ["ast"],
+          "*": ["abi", "metadata", "devdoc", "userdoc", "storageLayout", "evm.legacyAssembly", "evm.bytecode", "evm.deployedBytecode", "evm.methodIdentifiers", "evm.gasEstimates", "evm.assembly"]
+        }
+    },
+    "remappings":[
+      "@openzeppelin/contracts/access/Ownable.sol=https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.5.0/contracts/access/Ownable.sol",
+      "@openzeppelin/contracts/token/ERC20/=https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.5.0/contracts/token/ERC20/",
+      "@bob-collective/bitcoin-spv/=https://github.com/bob-collective/bitcoin-spv/blob/master/src/",
+      "@bob-collective/bob/=https://github.com/bob-collective/bob/blob/master/src/"
+    ]
+  }
+}
+```
+Enter the below code in `HelloBitcoin.sol` file. To learn more about the contract [checkout the Solidity tutorial guide](https://docs.soliditylang.org/en/v0.8.21/introduction-to-smart-contracts.html#subcurrency-example).
+
+```solidity
```

Comment:
Could you please add solidity in the theme config so that the syntax highlighting works? See docs: https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages